packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,27 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.addEventListener('opened-changed', () => {
+          contextMenu.items = contextMenuItems;
+          const listBox = document.querySelector('vaadin-context-menu-list-box');
+          setTimeout(() => {
+            const { width } = getComputedStyle(listBox);
+            if (menuItem.description) {
+              const el = document.createElement('vaadin-context-menu-item');
+              el.style.minWidth = '256px';
+              el.style.maxWidth = width;
+              el.classList.add('description');
+              render(html` ${unsafeHTML(menuItem.description)} `, el);
+              contextMenu.items = [
+                ...contextMenuItems,
+                { component: document.createElement('hr') },
+                { component: el },
+              ];
+              contextMenu.render();
```

Comment:
Hm, we manually render now?

Reply:
Without `contextMenu.render()` the description isn't added. I believe context menu does it's calculations before the `opened-changed` event.